astroquery/alma/core.py

```diff
@@ -306,26 +295,6 @@ def stage_data(self, uids):
                                                     ))
         tbl = self._json_summary_to_table(json_data, base_url=base_url)
 
-        # staging_root = BeautifulSoup(data_page.content)
-        # downloadFileURL = staging_root.find('form').attrs['action']
-        # data_list_url = os.path.split(downloadFileURL)[0]
-
-        # # Old version, unreliable: data_list_url = staging_submission.url
-        # log.debug("Data list URL: {0}".format(data_list_url))
-        # self._staging_log['data_list_url'] = data_list_url
-
-        # time.sleep(1)
-        # data_list_page = self._request('GET', data_list_url, cache=True)
-        # self._staging_log['data_list_page'] = data_list_page
-        # data_list_page.raise_for_status()
-
-        # if 'Error' in data_list_page.text:
-        #     errormessage = staging_root.find(
-        #         'div', id='errorContent').string.strip()
-        #     raise RemoteServiceError(errormessage)
-
-        # tbl = self._parse_staging_request_page(data_list_page)
-
```

Comment:
This was an old implementation.  Could you remove this in a separate commit, stating "Remove commented out old ALMA staging implementation"?  Again, it may be needed in the future.